biz.aQute.bnd.remote.junit/src/aQute/bnd/remote/junit/BundleSpecBuilder.java

```diff
@@ -205,55 +237,73 @@ public String fragmenthost() {
 	 * Extension options for the Fragment-Host
 	 */
 	enum Extension {
-		framework, bootclasspath
+		FRAMEWORK("framework"),
+		BOOTCLASSPATH("bootclasspath");
+
+		public final String value;
```

Comment:
Don't make this public. Make it private, override toString and have it return value.

Reply:
Good idea